src/App.css

```diff
@@ -85,4 +85,52 @@
     color: #ccc !important;
     background: none;
   }
+
+  from { transform: rotate(0deg); }
+  to { transform: rotate(360deg); }
+}
+
+p{
```

Comment:
How about creating a class to target this particular element instead of implementing the style for all `p` elements?

Reply:
Thank you. Let me work on that.

Reply:
Awesome 👍